x-pack/heartbeat/Jenkinsfile.yml

```diff
@@ -0,0 +1,18 @@
+when:
```

Comment:
This is a new step, though there is no build in place yet but the packaging itself.